content/hi/cloud-native-security.md

```diff
@@ -0,0 +1,18 @@
+---
+title: क्लाउड नेटिव सुरक्षा (Cloud Native Security)
+status: Feedback Appreciated
+category: संकल्पना
+tags:  ["सुरक्षा"]
+---
+
+## यह क्या है
+
+क्लाउड नेटिव सुरक्षा एक दृष्टिकोण है जो क्लाउड नेटिव अनुप्रयोगों में सुरक्षा निर्मित करता है। यह सुनिश्चित करता है कि सुरक्षा विकास से लेकर उत्पादन तक संपूर्ण अनुप्रयोग जीवनचक्र का हिस्सा बनी रहे। क्लाउड नेटिव सुरक्षा पारंपरिक सुरक्षा प्रतिरूप के समान मानकों को क्लाउड नेटिव वातावरण के विवरणों, अर्थात् तेज़ी से कोड परिवर्तन और अत्यधिक अल्पकालिक अवसरंचनाओं  को अपनाते हुए सुनिश्चित करने का प्रयास करती है । क्लाउड नेटिव सुरक्षा डेवसेकॉप्स [DevSecOps](/devsecops/) नामक अभ्यास से अत्यधिक संबंधित है।
```

Comment:
1. replace उत्पादन with निर्माण
2.  यह सुनिश्चित करता है कि विकास से लेकर उत्पादन तक, सुरक्षा संपूर्ण अनुप्रयोग जीवनचक्र का हिस्सा बनी रहे।

Reply:
Thanks for reviewing the PR, @sayantani11. I've made the changes in line 10, please have a look!